src/geowombat/core/api.py

```diff
@@ -513,6 +513,18 @@ def __init__(
                         w = src.block_window(1, 0, 0)
                         kwargs["chunks"] = (band_chunks, w.height, w.width)
 
+                # kw = {

```

Comment:
I would like to send a note to the user if they have config updates when using mosaic or stacking multiple images - since updates to res, etc don't get used since open_warp isn't called -  

However when `_check_config_globals` is run its breaking all the tests and I am not sure why.